modules/calloutparsers/ocallouts/ocallouts.py

```diff
@@ -31,6 +31,10 @@
     "BMC0007": [
         "A system data mismatch has been detected."
     ]
+
+    "BMC0008": [
+        "Failed parts present in the system need service."
```

Comment:
nit: maybe break up the sentence like:
    Failed parts present in the system - service needed.
    Failed parts present in the system.   Service needed

Reply:
done